test/controllers/predicates/AuthenticationPredicateSpec.scala

```diff
@@ -174,5 +174,16 @@ class AuthenticationPredicateSpec extends TestSupport with MockAuthenticationPre
       }
 
     }
+
+    "called with a confidence level of 250" should {
+      "return Ok (200)" in {
+        enable(IvUplift)
```

Comment:
I would assume this is not required ? so, ie. if IvUplift is not enabled we still should be able to login with CL250 (np);
but in case we are trying with CL200 and IvUplift is on we are redirected (so we are testing redirect etc)